web/package.json

```diff
@@ -10,8 +10,10 @@
     "@types/node": "^16.11.22",
     "@types/react": "^17.0.39",
     "@types/react-dom": "^17.0.11",
+    "package.json": "^2.0.1",
```

Comment:
what is the purpose of this package? it seems to be deprecated on [npm](https://www.npmjs.com/package/package.json).